### ISSUE TITLE
Testing config resolution at task level

### ISSUE THREAD

**Issue by ewels** [askimed/nf-test#60]
Hi there,

Thanks for nf-test, having a lot of fun with it in @nf-core and it's working well. Apologies if this is a naive question, still getting to grips with the concepts.

In the pipeline I'm working on at the moment I have a particularly [horrendous chunk](https://github.com/nf-core/methylseq/blob/977767c45496e12d49e1798d89255698849bb46e/conf/modules.config#L53-L139) of Nextflow config and I'd like to simply test that the config resolution works in the way that I expect. In other words, when I provide _these_ params, I want to check that `process.withName: TRIMGALORE { ext.args }` contains _this_ substring. It would also be fine to check that the generated `.command.sh` contains a certain substring.

Is there an existing way to dig into the resolved task details like this?

Phil

**Comment by lukfor** [comment]
Good point, thanks for the suggestion! ATM there is no way to check this, but I keep this task on our todo-list 👍
